ui/app/AppLayouts/Communities/popups/FinaliseOwnershipDeclinePopup.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQml.Models 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    // Community related props:
+    required property string communityName
+
+    signal cancelClicked
```

Comment:
Need to handle that signal? It just closes the popup w/o doing anything right

Reply:
Removed!